scripts/format-queries.lua

```diff
@@ -1,12 +1,18 @@
--- #!/usr/bin/env -S nvim -l
+#!/usr/bin/env -S nvim -l
 
 local ts = vim.treesitter
 local get_node_text = ts.get_node_text
 local root_path = vim.fn.fnamemodify(string.sub(debug.getinfo(1, 'S').source, 2), ':p:h:h')
 
-vim.opt.runtimepath:append(root_path)
-vim.cmd.runtime({ 'plugin/nvim-treesitter.lua', bang = true })
-vim.cmd.runtime({ 'plugin/query_predicates.lua', bang = true })
+ts.query.add_predicate('has-type?', function(match, _, _, pred)
+  local node = match[pred[2]]
+  if not node then
+    return true
+  end
+
+  local types = { unpack(pred, 3) }
+  return vim.list_contains(types, node:type())
+end, true)
```

Comment:
I'm also using `not-has-parent`, to achieve the collapsing opening parentheses

Reply:
That one is already in Neovim core (in 0.10, which we're targeting here).